instant-xml-macros/src/common.rs

```diff
@@ -0,0 +1,43 @@
+use syn::{DeriveInput, Lit, Meta, NestedMeta};
+
+pub fn retrieve_default_namespace(input: &DeriveInput) -> Option<String> {
+    for item in get_meta_items(&input.attrs) {
+        match item {
+            NestedMeta::Meta(Meta::List(list)) => {
+                if let Some(id) = list.path.get_ident() {
+                    if id == "namespace" {
+                        let nested = list.nested.first().unwrap();
+                        match nested {
+                            NestedMeta::Lit(Lit::Str(v)) => {
+                                let value: syn::Path = v.parse().unwrap();
```

Comment:
Why are we parsing a `Path` out of a `Lit::Str` here?

Reply:
It was leftover after something, sorry for this PR being a little untidy. I was in rush, but at the same time I wanted to open it yesterday, to gather your thoughts.